dir.props

```diff
@@ -186,16 +192,16 @@
   <PropertyGroup Condition="'$(NetCoreSurface)' != 'true'">
     <DefineConstants>$(DefineConstants);FEATURE_APARTMENT_STATE</DefineConstants>
     <DefineConstants>$(DefineConstants);FEATURE_APM</DefineConstants>
-    <DefineConstants>$(DefineConstants);FEATURE_APPDOMAIN</DefineConstants>
-    <FeatureAppDomain>true</FeatureAppDomain>
+    <DefineConstants Condition="'$(MonoBuild)' != 'true'">$(DefineConstants);FEATURE_APPDOMAIN</DefineConstants>
+    <FeatureAppDomain Condition="'$(MonoBuild)' != 'true'">true</FeatureAppDomain>
```

Comment:
Hmm.  I don't love this mixing--should we have a separate block for the Mono defines?

On the other hand, they should be *mostly* the same as the full-framework defines, and I don't love having to double-list everything, either . . .